internal/util/csiconfig.go

```diff
@@ -45,6 +45,7 @@ const (
 	"clusterID": "<cluster-id>",
 	"rbd": {
 		"radosNamespace": "<rados-namespace>"
+		"mirrorDaemonCount": "1"
```

Comment:
Great test case, but unfortunately missing in the unit tests.

Please add a test-case that expects an error when the argument is a number as `"string"` (maybe convert it?) or even an invalid number.